src/blocks/common.blocks/block-example/_view/block-example_view_inline.bemtree

```diff
@@ -23,7 +23,7 @@ block('block-example').mod('view', 'inline')(
                         {
                             elem: 'source-code',
                             elemMods: { view: 'text' },
-                            content: this._bemjson
+                            content: this._bemjson.replace(/</g, '&lt;').replace(/>/g, '&gt;')
```

Comment:
I suggest to use [strings](https://ru.bem.info/libs/bem-core/v2.5.1/desktop/strings/) here.